### PR TITLE
overturemaps: init at 0.9.0

### PR DESCRIPTION
**Pull request by crimeminister**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: maintainers/maintainer-list.nix
Comment (eclairevoyant) [inline maintainers/maintainer-list.nix:4235]: please split this out into a separate initial commit `maintainers: add crimeminister`

File: pkgs/tools/misc/overturemaps/default.nix
Comment (eclairevoyant) [inline pkgs/tools/misc/overturemaps/default.nix:7]:
```suggestion
 python3Packages,
 fetchFromGitHub,
```
never pass `pkgs` to a package

File: pkgs/tools/misc/overturemaps/default.nix
Comment (eclairevoyant) [inline pkgs/tools/misc/overturemaps/default.nix:10]:
```suggestion
python3Packages.buildPythonPackage rec {
```

File: pkgs/tools/misc/overturemaps/default.nix
Comment (eclairevoyant) [inline pkgs/tools/misc/overturemaps/default.nix:22]:
```suggestion
  build-system = with python3Packages; [ poetry-core ];
```

File: pkgs/tools/misc/overturemaps/default.nix
Comment (eclairevoyant) [inline pkgs/tools/misc/overturemaps/default.nix:24]:
```suggestion
  dependencies = with python3Packages; [
```

File: pkgs/top-level/all-packages.nix
Comment (eclairevoyant) [inline pkgs/top-level/all-packages.nix:11174]:
```suggestion
```

and please move the package expression to the `pkgs/by-name` structure

**Review by eclairevoyant — CHANGES_REQUESTED** [review]
Thanks for the contribution! As per guidelines, adding your maintainer entry to maintainer-list should be one commit, and the package init should fall under one commit. Some other feedback below:

File: pkgs/tools/misc/overturemaps/default.nix
Comment (eclairevoyant) [inline pkgs/tools/misc/overturemaps/default.nix:20]: any reason we don't just fetch from pypi instead? Since there's no tagged releases on GH.

Comment (crimeminister) [reply pkgs/tools/misc/overturemaps/default.nix]: I read some advice somewhere to prefer the option that allows for the possibility of running tests, often included in a source repo and not in packaged releases. Happy to change this though.

File: maintainers/maintainer-list.nix
Comment (crimeminister) [reply maintainers/maintainer-list.nix:4235]: Fixed!

**Comment by crimeminister** [comment]
Thanks for the feedback @eclairevoyant.

**Review by eclairevoyant — COMMENTED** [review]
Seems to run correctly, I can approve once the last commit is squashed into the prior

**Review by eclairevoyant — APPROVED** [review]

**Pull request merged**
